Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs

```diff
@@ -254,6 +256,11 @@ private void OnComponentInit(EntityUid uid, NukeOperativeComponent component, Co
         }
     }
 
+    private void OnAmnesiaEventAfter(AmnesiaEventAfter ev)
+    {
+        RemCompDeferred<NukeOperativeComponent>(ev.Target);
+    }
+
```

Comment:
Antags shouldn't have to hardcode checks in amnesia, amnesia should handle any mind changes.